server/container/guice/memory-guice/src/main/java/org/apache/james/modules/data/MemoryDataModule.java

```diff
@@ -54,7 +54,7 @@
 
 public class MemoryDataModule extends AbstractModule {
     private static final MailRepositoryStoreConfiguration.Item FILE_MAILREPOSITORY_DEFAULT_DECLARATION = new MailRepositoryStoreConfiguration.Item(
-        ImmutableList.of(new Protocol("file")),
+        ImmutableList.of(new Protocol("file"), new Protocol("memory")),
```

Comment:
why do you need two protocols ?

Reply:
I hopped that we have a Memory protocol, it seems not